wqflask/wqflask/templates/show_trait.html

```diff
@@ -352,16 +353,15 @@ <h3 class="panel-title">
                   'order': [[1, "asc"]],
                   'sDom': "Ztr",
                   'autoWidth': true,
-                  'deferRender': true,
-                  'paging': false,
                   'orderClasses': true,
-                  'scrollY': "600px",
-                  'scrollCollapse': false,
-                  'processing': true,
-                  'language': {
-                    'loadingRecords': '&nbsp;',
-                    'processing': 'Loading...'
-                  }
+                  "scrollY": "50vh",
+                  'scroller':  true,
+                  'scrollCollapse': true
+                  // 'processing': true,
+                  // 'language': {
+                  //   'loadingRecords': '&nbsp;',
+                  //   'processing': 'Loading...'
+                  // }
```

Comment:
**suggestion: nitpick:** It's a good idea to remove unnecessary comments. This class of comments--commented out code-- after a while becomes "dead code" that either usually ends either being clutter or outright misleading.